isis/src/base/apps/framestitch/framestitch.cpp

```diff
@@ -0,0 +1,220 @@
+#include "framestitch.h"
+
+#include <QString>
+
+#include "Brick.h"
+#include "Buffer.h"
+#include "Cube.h"
+#include "CubeAttribute.h"
+#include "Camera.h"
+#include "Histogram.h"
+#include "IException.h"
+#include "IString.h"
+#include "LineManager.h"
+#include "ProcessByBrick.h"
+#include "PvlGroup.h"
+#include "PvlKeyword.h"
+#include "Statistics.h"
+#include "UserInterface.h"
+
+using namespace std;
+using namespace Isis;
+
+namespace Isis {
+
+  /**
+   * Helper function to count the size of frames in a cube
```

Comment:
It looks that the comment is not aligned with what the function does. The function appears to find the median framelet size rather than counting. The function name may also need to be renamed to computeMedianFrameSize() or computeMedianFrameletSize(), and somewhere it should be explained what a framelet is.

I think this computes in fact framelet height, rather than its 2D size.